vendor.mod

```diff
@@ -0,0 +1,5 @@
+module github.com/docker/go-events
+
+go 1.13
+
+require github.com/sirupsen/logrus v1.9.3
```

Comment:
We should indeed look if we can get rid of the log dependency.

It looks like it's currently using "structured" logs (`WithField()`), otherwise we could've just swapped for a plain `log`.

Of course, ideally, the logger would be configurable / opt-in, but we don't pass a context, so we can't use a context-logger. But of course we could still define some basic interface and some utility to set the logger to use.

Either way; all for later concern (not this PR).

Reply:
+1, let me create an issue on this. One (of the few) log lines brought me to this repo and spurred my interest in it. There are some options available here each with its own trade-offs. New interfaces with context for passing logger and streamlined go-routine management, providing a built-in sink for logging async write errors, or something else I haven't thought yet.

Reply:
@thaJeztah, I created https://github.com/docker/go-events/issues/36 with some of my thoughts on this so far. Feel free to beat on it. :)